### ISSUE TITLE
Overriding color constants does not override all blocks with given color

### ISSUE THREAD

**Issue by AnmAtAnm** [google/blockly#1648]
### Problem statement

Overriding color constants does not override all blocks with given color.

### Expected Behavior

Overriding a Blockly color constant after loading the `blocks_compressed.js` and before constructing a block (ideally before injecting Blockly) should affect all blocks using the color constant.

### Actual Behavior

Several block types have the incorrect color, at least among the text blocks.

### Steps to Reproduce

1. Add the following lines to the top of the first `<script>` in the Playground (right after `'use strict'`):

```
Blockly.Msg.TEXTS_HUE = "#193863";  // dark blue
```

2. Load the Playground and open the text blocks toolbox category. Note the mix of block colors:

<img width="647" alt="screen shot 2018-03-29 at 10 59 01 am" src="https://user-images.githubusercontent.com/9916202/38105198-41c1fe32-3340-11e8-8c69-5e09d28be839.png">

### Operating System and Browser

 * Desktop Chrome

**Comment by CleoQc** [comment]
For historical purposes, this used to work with the code from a year ago. I don't know when the bug was introduced but it has been working for us before. 

Thanks for creating the issue for me.

Cleo

**Comment by swdunningASU** [comment]
After trying to change the color of the Loops and Conditionals and not being successful I was told to go here.  This is also happening with me.  The blocks look almost exactly the same, first half are green and second half is blue.

**Comment by swdunningASU** [comment]
Also, this only happens if I add both lines of code to the "start" function, only adding the first line of code does nothing and adding them to the <script> tag does nothing either....Although, I am not sure I am doing it correctly.

**Comment by AnmAtAnm** [comment]
The correct form is:

```HTML
<script type="text/javascript">
Blockly.Constants.Text.HUE = "#193863";  // dark blue
Blockly.Blocks.texts.HUE = Blockly.Constants.Text.HUE;  // dark blue
</script>
```

**Comment by AnmAtAnm** [comment]
I incorrectly wrote the bug referring to the old method of setting the color.  Since the move to JSON required message references, I've updated the above bug report and comments to hint at a preference of updating the `MSG` constant, instead of the prior `HUE` constant.

A PR with all blocks using the MSG constants is forthcoming.

**Comment by AnmAtAnm** [comment]
Just for clarification, the original version of this bug report suggested `Blockly.Constants.Text.HUE = "#193863";  // dark blue` was the correct form. This is the form that CleoQc suggested worked one year ago.

This was replaced (unfortunately without marking it as deprecated) by the Msg constants, so they could be used in JSON without making the direct JavaScript variable references (critical for the Android and iOS implementations).

This also has the benefit of adding a level of indirection that unbinds load ordering constraints between the block definitions and the constant values. The message dereferencing occurs at block creation time instead of at JavaScript/JSON load time.  PR #1749 will ensure all predefined blocks use the Msg constants at all locations.

**Comment by rachel-fenichel** [comment]
Fixed by #1749.
